financial-connections/src/test/java/com/stripe/android/financialconnections/features/success/SuccessViewModelTest.kt

```diff
@@ -95,17 +94,4 @@ internal class SuccessViewModelTest {
             assertThat(awaitItem()).isEqualTo(Complete())
         }
     }
-
-    private fun fakeSuccessContentRepository(
-        initialState: SuccessContentRepository.State = SuccessContentRepository.State(
```

Comment:
I apparently didn’t really think when adding this. It elegantly hid the issue that broke the flow 🙃